deepchecks/vision/utils/vision_properties.py

```diff
@@ -130,3 +130,45 @@ def static_prop_to_cache_format(static_props: STATIC_PROPERTIES_FORMAT) -> PROPE
                 props_cache[input_type][prop_name] = prop_vals
 
     return props_cache
+
+
+def static_properties_from_df(df,
+                              image_cols: Tuple = (),
+                              label_cols: Tuple = (),
+                              prediction_cols: Tuple = (),
+                              partial_image_cols: Tuple = ()) -> STATIC_PROPERTIES_FORMAT:
+    """
+    Transform the precalculated properties to the static properties format.
```

Comment:
```suggestion
    Transform the pre-calculated properties from a DataFrame to the expected dict format.
    
    Read more about the excepted dict format for pre-calculated properties at X
```
Replace X with link to your docs